HerPortal.UnitTests/Builders/UserBuilder.cs

```diff
@@ -29,12 +29,6 @@ public UserBuilder WithLocalAuthorities(List<LocalAuthority> localAuthorities)
         user.LocalAuthorities = localAuthorities;
         return this;
     }
-    
-    public UserBuilder WithConsortia(List<Consortium> consortia)
-    {
-        user.Consortia = consortia;
-        return this;
-    }
```

Comment:
Just not used? Should it be?

Reply:
the same function appears below; this is a duplicate